reloj-checador/app/controllers/attendaces_controller.rb

```diff
@@ -1,6 +1,6 @@
-class AttendacesController < ApplicationController
+class AttendacesController < CheckinsController
```

Comment:
Why do we need to inherit from `ChekinsController`?

Reply:
ApplicationController is for admin handling (with auth) and CheckinsController is used by employees to register their check in or check out (No auth needed). So for secure reasons and to prevent auth issue we use CheckinsController as a separete controller inheriting from ActionController::Base